hyperspy/_components/eels_cl_edge.py

```diff
@@ -93,6 +93,9 @@ def __init__(self, element_subshell, GOS=None):
                             'fine_structure_coeff',
                             'effective_angle',
                             'onset_energy'])
+
+        self._whitelist['element_subshell'] = ('init', element_subshell)
+        self._whitelist['GOS'] = ('init', GOS)
```

Comment:
If you don't mind it I'll keep on commenting on this ones, but please
fix the issues that belong to #167  in #167 and we'll make sure that we
merge that one first, or is it not a good idea?
On Fri, 2015-11-06 at 07:47 -0800, Tomas Ostasevicius wrote:
> In hyperspy/_components/eels_cl_edge.py:
> > @@ -93,6 +93,9 @@ def __init__(self, element_subshell, GOS=None):
> >                              'fine_structure_coeff',
> >                              'effective_angle',
> >                              'onset_energy'])
> > +
> > +        self._whitelist['element_subshell'] = ('init',
> element_subshell)
> > +        self._whitelist['GOS'] = ('init', GOS)
> It looks like it should. fine_structure_width as well, don't know why
> I did not add them. Will add to #167 , where it belongs. By the way,
> should I PR that again (to get rid of the old discussions), or is it
> fine like that?
> —
> Reply to this email directly or view it on GitHub.